cernopendata/config.py

```diff
@@ -37,7 +37,10 @@
 from cernopendata.modules.theme.config import *
 
 # Debug
-DEBUG = True
+DEBUG = os.environ.get(
+    'DEBUG',
+    'False'
+)
 TEMPLATES_AUTO_RELOAD = True
```

Comment:
I'd also treat `TEMPLATES_AUTO_RELOAD` as a debugging-related variable, i.e. when the `DEBUG` environment variable is True, then both `config.DEBUG` and `config.TEMPLATES_AUTO_RELOAD` are True; otherwise both are False by default.